server/src/main/java/org/elasticsearch/action/StatsLevel.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action;
+
+import org.elasticsearch.common.Strings;
+
+import static org.elasticsearch.action.ValidateActions.addValidationError;
+
+public enum StatsLevel {
+    CLUSTER("cluster"),
+    NODE("node"),
+    INDICES("indices"),
+    SHARDS("shards");
+
+    private final String name;
+
+    StatsLevel(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public static ActionRequestValidationException clusterLevelsValidation(final String level) {
```

Comment:
If we require a valid `Level` (instead of a `String`) then there's no need for this kind of validation at all.